includes/data-stores/class-wc-shipping-zone-data-store-table.php

```diff
@@ -0,0 +1,284 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * WC Shipping Zone Data Store: Custom Table.
+ *
+ * @version  2.7.0
+ * @category Class
+ * @author   WooCommerce
+ */
+class WC_Shipping_Zone_Data_Store_Table implements WC_Shipping_Zone_Data_Store_Interface, WC_Object_Data_Store {
```

Comment:
I think this is another like customers - _table isn't really needed because there isn't an alternative?